wrappers/nodejs/src/addon.cpp

```diff
@@ -2385,20 +2392,51 @@ class RSContext : public Nan::ObjectWrap {
   }
 
   static void New(const Nan::FunctionCallbackInfo<v8::Value>& info) {
-    if (info.IsConstructCall()) {
-      RSContext* obj = new RSContext();
-      obj->Wrap(info.This());
-      info.GetReturnValue().Set(info.This());
-    }
+    if (!info.IsConstructCall())
+      return;
+
+    ContextType type = kNormal;
+    if (info.Length()) {
+      v8::String::Utf8Value type_str(info[0]->ToString());
+      if (!strcmp(*type_str, "recording"))
```

Comment:
Please avoid using `strcmp`

Reply:
Yes, we can use the compare() method of std::string instead.